main.py

```diff
@@ -0,0 +1,49 @@
+import ssl
+from time import time
+from datetime import datetime
+from urllib.parse import urlparse
+from pathlib import Path
+import telebot
+
+# access token for telegram bot
+access_token = ''
+# the time to notify of the expiration of the certificate   
+notice_date = 432000
+bot = telebot.TeleBot(access_token)
+
+
+def check_certificate(url):
+    parsed_url = urlparse(url)
+    try:
+        cert = ssl.get_server_certificate(
+            (parsed_url.hostname, parsed_url.port or 443))
+    except Exception:
+        return "Bad request"
+    # save cert to temporary file (filename required for _test_decode_cert())
+    temp_filename = Path(__file__).parent / "temp.crt"
```

Comment:
Is there any way to decode certificate in memory (without creation of additional files)? User can face the issue when he doesn't have permissions to write in current folder